services/web-app/components/nav-tree/nav-tree.js

```diff
@@ -55,6 +55,15 @@ const NavTree = ({ activeItem, items = [], label }) => {
     setItemNodes(newItemNodeArray)
   }, [items])
 
+  useEffect(() => {
+    requestAnimationFrame(() => {
+      // Remove tabstop from headings
+      document
+        .querySelectorAll('[class*=section-group]')
+        .forEach((heading) => heading.setAttribute('tabindex', '-1'))
+    })
+  }, [])
+
```

Comment:
The `h2`s inside the treeview where showing up with `tabIndex=0` and I couldn't figure out why, my theory is it's the treeview doing it behind the scenes and this is the only way I could figure out to override them to remove that hidden tabStop, open to any suggestions though...

Reply:
leaving this as-is hehe